Quasardb/GetBytes.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Quasardb
+{
+    /// <summary>
+    /// Helper class
+    /// </summary>
+    public static class Helper
+    {
+        /// <summary>
+        /// Safely Get Bytes
+        /// </summary>
+        public static byte[] GetBytes(int size, IntPtr Pointer)
```

Comment:
Don't you rather want to pass a `size_t` and cast here?

Reply:
Converted to Uint64 which seems to be the standard for size_t.